### ISSUE TITLE
Unmerged PRs are in a messy state

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-at#624]
There are currently several open pull requests, that fix problems reported by PAC during the test writing process.  Unfortunately, none of them have been merged, meaning that the problems are still present.  Additionally, they all have many merge conflicts, meaning that merging them is not straight forward.

PAC will not be expending energy on:

* fixing the conflicts, which are not in our PRs; nor
* fixing tests without clear error messages that the pull requests provide access to, because trying to work around problems that already have fixes is a waste of everybody's time.

@s3ththompson Please arrange for the following PRs to be merged as a priority, ideally this week:

* https://github.com/w3c/aria-at/pull/608
* https://github.com/w3c/aria-at/pull/610
* https://github.com/w3c/aria-at/pull/612
* https://github.com/w3c/aria-at/pull/613
* https://github.com/w3c/aria-at/pull/615
* https://github.com/w3c/aria-at/pull/618

CC @SinaBahram / @mcking65 / @howard-e / @alflennik / @IsaDC

**Comment by jscholes** [comment]
While trying to work on https://github.com/w3c/aria-at/pull/623, we encountered errors.  These were non-descript, as already described in previous issues.  Now that certain PRs have been merged while https://github.com/w3c/aria-at/pull/623 is still in-flight, `master` cannot be merged into that PR because of merge conflicts caused by the `build` directory being totally removed.  Please also arrange for this to be fixed, so that we can merge `master` into https://github.com/w3c/aria-at/pull/623 and benefit from the referenced pull requests being merged.

**Comment by jesdaigle** [comment]
@jscholes My team has these items as well as [issue 621](https://github.com/w3c/aria-at/pull/621) in the next sprint, which begins tomorrow and ends March 3rd,  to resolve. 5 of the 6 issues referenced require one more reviewer, which I will be assisting in. Please let me know if you have any questions.

**Comment by jscholes** [comment]
@jesdaigle Thanks, but I'm not sure what that implies in terms of a date when the open PRs will be reviewed and merged.  February 17 to March 3 is a significant window.  Can you please be more specific to allow us to plan our work?

**Comment by jesdaigle** [comment]
@jscholes I am averse to promising specific dates outside of sprint windows. We will certainly prioritize these and get them resolved as soon as possible, at which time you will be notified. I've already reviewed 1 of the 5 that needs a second reviewer and will get to the rest as quickly as possible. Ideally, I'd like to resolve [issue 621](https://github.com/w3c/aria-at/pull/621) first as that will clear the path to addressing the many conflicts from generated files in the PR's that need to be merged. I will be looking at that at tomorrow. Thank you for understanding.

**Comment by jscholes** [comment]
@jesdaigle These pull requests will significantly unblock and accelerate the test writing process, which has been stalled for some time owing to the presence of the issues that they resolve.  We are, therefore, grateful to members of the Bocoup team for the development work, and the code reviews and approvals so far.

However, the proposed sprint window is not sufficiently granular to inform our scheduling, and allow us to make a determination as to our ability to ramp up the test writing process for 2022 (including the completion of 2021 work that was delayed by tooling issues).  Particularly given the fact that in some cases, work was commenced on the pull requests almost a month ago.

With this context in mind, we will plan to work through the open pull requests, and merge those with at least one approval from a Bocoup team member, by 14:00 Eastern this Friday (February 18), including any clean up of merge conflicts/necessary rebasing.  If there are reasons as to why a given PR will not be in a merge-ready state by that time, other than additional reviews beyond those already provided and/or the presence of merge conflicts, please enumerate those reasons either in this issue, or on the relevant PRs themselves, so we can avoid merging in-progress work.  I assume that all PRs with at least one approving review are considered complete, otherwise the current reviews would themselves be inaccurate and inappropriate.

Furthermore, we propose that going forward, pull requests in this specific repo that relate to test tooling should be merged after an approving review from at least one member of both the Bocoup and PAC teams.  The alternative to this is that Bocoup commit to sharing more specific dates/prioritisation info ahead of time.  Further discussion on this topic will take place between PAC, Bocoup and Meta on our next sync-up meeting(s).

**Comment by jesdaigle** [comment]
@jscholes as previously stated we have prioritized these and will get through them as quickly as possible. They are on deck for the sprint that begins tomorrow and prioritized to address. I apologize if you find that insufficient.

**Comment by jscholes** [comment]
@jesdaigle Are they top of the pile in that sprint?  I.e. is there anything prioritised before them?

**Comment by jscholes** [comment]
Also, what is the scope of a sprint in this context?  Is the sprint window shared across all of Bocoup's ARIA-AT input, including ARIA-AT App?  Or are there specific sprints broken down by workstream?  Trying to understand the implications of the approach on our work.  Thank you in advance for any additional input you can provide.

**Comment by jesdaigle** [comment]
Our sprints are comprised of all workstreams with team members working concurrently on the pieces they are assigned. I provide project support across all workstreams.
